### PR TITLE
B/create drop user stmt

### PR DESCRIPTION
**Pull request by bogensberger**

### CODE REVIEW

File: sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java [hunk 1]
```diff
@@ -368,6 +368,18 @@ public void testCreateAnalyzerStmtBuilder() {
     }
 
     @Test
+    public void testCreateUserStmtBuilder() {
+        printStatement("create user \"Günter\"");
```
Comment (chaudum) [inline sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java]: can you also try whitespace

File: sql/src/main/java/io/crate/analyze/CreateUserAnalyzedStatement.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.analyze;
+
+public class CreateUserAnalyzedStatement implements DDLStatement {
+
+    private final String name;
```
Comment (chaudum) [inline sql/src/main/java/io/crate/analyze/CreateUserAnalyzedStatement.java]: personal preference: name the field `username`, bc it's more descriptive

File: sql/src/main/java/io/crate/analyze/DropUserAnalyzedStatement.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.analyze;
+
+public class DropUserAnalyzedStatement implements DDLStatement {
+
+    private final String name;
```
Comment (chaudum) [inline sql/src/main/java/io/crate/analyze/DropUserAnalyzedStatement.java]: personal preference: name the field `username`, bc it's more descriptive

File: sql-parser/src/main/antlr/SqlBase.g4 [hunk 4]
```diff
@@ -755,6 +757,8 @@ ALWAYS: 'ALWAYS';
 
 READ: 'READ';
 
+USER: 'USER';
```
Comment (chaudum) [inline sql-parser/src/main/antlr/SqlBase.g4]: is `USER` an new reserved keyword? if so, it needs to be documented in breaking changes

Comment (bogensberger) [inline sql-parser/src/main/antlr/SqlBase.g4]: added it to the "non reserved keywords", as User is a highly used word

**Pull request merged**